ctx.go

```diff
@@ -300,6 +301,59 @@ func (c *DefaultCtx) Route() *Route {
 	return c.route
 }
 
+// Matched returns true if the current request path was matched by the router.
+func (c *DefaultCtx) Matched() bool {
```

Comment:
can we replace https://github.com/gofiber/fiber/blob/2555a4f1b5f579cce0ec054056c5eb0b61a2e3f9/ctx.go#L546 with this

Reply:
or we just expose the GetMatched